Pipfile

```diff
@@ -17,6 +17,7 @@ quandl = "==3.4.8"
 statsmodels = "==0.10.1"
 bs4 = "==0.0.1"
 tweepy = "==3.5.0"
+pandas = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3"
```

Comment:
You don't need to change this version in the backend as this is going to be server side and should be up to date with python